libraries/fiff/fiff_coord_trans.cpp

```diff
@@ -162,6 +162,25 @@ bool FiffCoordTrans::read(QIODevice& p_IODevice, FiffCoordTrans& p_Trans)
 
 //=============================================================================================================
 
+void FiffCoordTrans::write(QIODevice &qIODevice)
```

Comment:
can `QIODevice &qIODevice` be declared as `const`? Depends on what `start_file()` asks for.

Reply:
start_file() does not take const QIODevices